server/homebrew.api.js

```diff
@@ -43,9 +43,12 @@ const getBrew = (accessType)=>{
 				}
 			});
 		stub = stub?.toObject();
+		if(stub?.authors && !stub?.authors.includes(req.account.username)) {
+			throw 'Current logged in user does not have access to this brew.';
+		}
```

Comment:
`getBrew` is also used to retrieve brews for sharing. This will block non-authors from viewing a share file.

Reply:
Just added an addition to this if to make sure the accessType is 'edit'!

Reply:
What happens if an anonymous user requests an edit URL? Does `[req.account.username]`produce an empty string or is it an invalid reference?

Reply:
Oop lemme check

Reply:
Yep that was an issue, fixed in https://github.com/naturalcrit/homebrewery/pull/2509/commits/6b8db74a2b51a17cbbac378ca99c0429f05f7aa3